playwright.config.ts

```diff
@@ -4,9 +4,9 @@ import { defineConfig, devices } from "@playwright/test";
  * Read environment variables from file.
  * https://github.com/motdotla/dotenv
  */
-// import dotenv from "dotenv";
-// import path from "path";
-// dotenv.config({ path: path.resolve(__dirname, ".env") });
+import dotenv from "dotenv";
+import path from "path";
+dotenv.config({ path: path.resolve(__dirname, ".env.local") });
```

Comment:
I don't think we need or want this (that is, reading in from `.env.local` all the time). Instead, we are loading in the environment variables with Github actions here - https://github.com/NYPL/nypl-library-card-app/blob/main/.github/workflows/playwright-tests.yml#L26

Reply:
TIL: GH will always take precendee over the vars from what's set in the `yml` file . So technically this is _OK_ ? I wonder what's the standard practice for reading the `env.*` vars in playwright @clarissarichard